CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.7.0-dev
 
+### Added
+
+- Support for `~/` at the beginning of configuration file imports
```

Comment:
What do you mean?

Reply:
I mean, users don't know that they can use `~/` to refer to home dir? It's not something intuitive on Windows, also we should mention that we don't support `~user` syntax.

Reply:
I've decided to use `~/` in the example path. Hopefully that should give people an idea of what is possible?

The rest people should be able to figure out just by trying things, I don't think a lot of people are going to try and use the `~user` syntax since it's rather pointless.

Of course we could add a full comment, but I feel like that would make things more difficult rather than being helpful?